### ISSUE TITLE
ddtbd.plugin

### ISSUE THREAD

**Issue by world0123** [winter2020/oo7#7]
when I run "bapbuild ddtbd.plugin",the error hapened.
error: no such file or directory.
then i file the ddtbd.plugin, it shows that broken symbolic link.
so what can I solve the problem?
Thank you in advance!

**Comment by ivg** [comment]
I don't think that this repository is actively monitored. I am the author of the static analysis part of this plugin. aka the `dtbd.plugin`, which is also available as a part of [bap-toolkit][1]. Feel free to drop by there, it is in the `spectre` folder and should be built without any problem in the modern infrastructure. 

[1]: https://github.com/BinaryAnalysisPlatform/bap-toolkit
